src/qt/forms/sendcoinsdialog.ui

```diff
@@ -850,7 +850,7 @@
                    <property name="toolTip">
                     <string>Specify a custom fee per kB (1,000 bytes) of the transaction's virtual size.
 
-Note:  Since the fee is calculated on a per-byte basis, a fee of "100 satoshis per kB" for a transaction size of 500 bytes (half of 1 kB) would ultimately yield a fee of only 50 satoshis.</string>
+Note:  Since the fee is calculated on a per-byte basis, a fee rate of "100 satoshis per kvB" for a transaction size of 500 virtual bytes (half of 1 kvB) would ultimately yield a fee of only 50 satoshis.</string>
```

Comment:
Why the addition of the word "rate"? I thought the word "rate" usually meant over time, like a frequency. Not sure why we need the word here, given that a fee is a one to one relationship to a TX.

Reply:
We're everywhere referring to fee/(v)byte as feerate.